src/Codeception/Lib/InnerBrowser.php

```diff
@@ -1054,7 +1054,7 @@ protected function getFormFor(Crawler $node): SymfonyForm
             $this->fail('The selected node is not a form and does not have a form ancestor.');
         }
 
-        $identifier = $form->attr('id') ?: $form->attr('action');
+        $identifier = $form->attr('id') ?: $form->attr('action') ?? '';
         if (!isset($this->forms[$identifier])) {
```

Comment:
Lower PHP versions already converted `null` array keys to an empty string (https://3v4l.org/5UbfM) implicitly.

However, using `null` is deprecated in PHP 8.5.
That's why an empty string is used explicitly.